Exercise_2/tcl_spack.sh

```diff
@@ -12,11 +12,11 @@
 source ~/spack/share/spack/setup-env.sh
```

Comment:
You will need to 
module load python/3.8.1

before your source the environment.

Reply:
You also might want to add this:

#####Use after training##############
##SBATCH -p express
#####################################

#####Use during training###############
#SBATCH --partition=reservation
#SBATCH --reservation=bootcamp_cpu_2023
#######################################

so it is consist with the other sessions. I just added it as well to my scripts.

Reply:
You still need to add `module load python/3.8.1` and remove the first `source ~/spack/share/spack/setup-env.sh`. I think this is just a typo. Thank you!!